saleor/dashboard/order/forms.py

```diff
@@ -1,27 +1,267 @@
 from django import forms
 from django.conf import settings
 from django.core.validators import MinValueValidator
-from django.urls import reverse_lazy
+from django.urls import reverse, reverse_lazy
 from django.utils.translation import npgettext_lazy, pgettext_lazy
 from django_prices.forms import MoneyField
 from payments import PaymentError, PaymentStatus
 
 from ...account.i18n import (
     AddressForm as StorefrontAddressForm, PossiblePhoneNumberFormField)
+from ...account.models import User
 from ...cart.forms import QuantityField
 from ...core.exceptions import InsufficientStock
-from ...discount.utils import decrease_voucher_usage
+from ...core.utils import ZERO_TAXED_MONEY
+from ...discount.models import Voucher
+from ...discount.utils import decrease_voucher_usage, increase_voucher_usage
+from ...order import CustomPaymentChoices, OrderStatus
 from ...order.emails import send_note_confirmation
-from ...order.models import Fulfillment, FulfillmentLine, OrderLine, OrderNote
+from ...order.models import (
+    Fulfillment, FulfillmentLine, Order, OrderLine, OrderNote, Payment)
 from ...order.utils import (
     add_variant_to_order, cancel_fulfillment, cancel_order,
     change_order_line_quantity, merge_duplicates_into_order_line,
     recalculate_order)
 from ...product.models import Product, ProductVariant, Stock
 from ...product.utils import allocate_stock, deallocate_stock
+from ...shipping.models import ANY_COUNTRY, ShippingMethodCountry
 from ..forms import AjaxSelect2ChoiceField
 from ..widgets import PhonePrefixWidget
-from .utils import fulfill_order_line
+from .utils import (
+    fulfill_order_line, get_voucher_discount_for_order,
+    update_order_with_user_addresses)
+
+
+class CreateOrderFromDraftForm(forms.ModelForm):
+    """Save draft order as a ready to fulfill."""
```

Comment:
I think we can `mark` as ready to fulfill, not `save`, also an article before `ready` is not necessary